test_scripts/LowVoltage/007_Low_Voltage_and_Wake_Up_absence_resumption_FULL_hmi_level_in_case_existed_app_in_full.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- 1. Mobile app is in FULL HMILevel
+-- 2. App has some persistent data that can be resumed
+-- 3. Mobile app is disconnected during the time frame of 30 sec (inclusive) before "SDL_LOW_VOLTAGE" unix signal from HMI
+-- 4. Mobile app registers with the same hashID during 30 sec. after "WAKE_UP" unix signal in the same ignition cycle
+-- 5. there is application currently in FULL
+-- SDL does:
+-- 1. not resume FULL HMILevel for app
+-- 2. resume persistent data
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/LowVoltage/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function addResumptionData()
+  common.rpcSend.AddCommand(1, 1)
+end
+
+local function checkResumptionData()
+  common.rpcCheck.AddCommand(1, 1)
+end
+
+local function checkResumptionHMILevel()
```

Comment:
@GetmanetsIrina In scripts for negative cases like this please also add check on absence of `BC.ActivateApp` request